Examples/SQLServerNetwork.ps1

```diff
@@ -0,0 +1,80 @@
+﻿Configuration SQLNetwork
+{
+    Import-DscResource -Module xSQLServer
+
+    # A Configuration expects at least one Node
+    Node $AllNodes.NodeName
+    {
+        # Set DCM Settings for each Node 
+        LocalConfigurationManager 
+        { 
+            DebugMode = "ForceModuleImport"
+            RebootNodeIfNeeded = $true 
+            ConfigurationMode = "ApplyOnly" 
+        } 
+
+        WindowsFeature "NET-Framework-Core"
+        {
+            Ensure = "Present"
+            Name = "NET-Framework-Core"
+        }
+
+        xSqlServerSetup "RDBMS"
+        {
+            DependsOn = @("[WindowsFeature]NET-Framework-Core")
+            SourcePath = $Node.SourcePath
+            SourceFolder = $Node.SQL2012FolderPath
+            InstanceName = $Node.Instance
+            Features = $Node.Features
+            SetupCredential = $Node.InstallerServiceAccount
+            SQLCollation = "Latin1_General_CI_AS"
+            SQLSysAdminAccounts = $Node.AdminAccount
+            SQLSvcAccount = $Node.LocalSystemAccount
+            AgtSvcAccount = $Node.LocalSystemAccount
+        }
+
+        xSqlServerFirewall "RDBMS"
+        {
+            DependsOn = @("[xSqlServerSetup]RDBMS")
+            SourcePath = $Node.SourcePath
+            SourceFolder = $Node.SQL2012FolderPath
+            InstanceName = $Node.Instance
+            Features = $Node.Features
+        }
+
+        # This will enable TCP/IP protocol and set custom static port, this will also restart sql service
+        xSQLServerNetwork "RDBMS"
+        {
+            DependsOn = @("[xSqlServerSetup]RDBMS")
+            InstanceName = $Node.Instance
+            ProtocolName = "tcp"
+            IsEnabled = $true
+            TCPPort = 4509
+            RestartService = $true 
+        }        
+    } 
+}
+
+$SecurePassword = ConvertTo-SecureString -String "Pass@word1" -AsPlainText -Force
+$InstallerServiceAccount = New-Object System.Management.Automation.PSCredential ("CONTOSO\!Installer", $SecurePassword)
+$LocalSystemAccount = New-Object System.Management.Automation.PSCredential ("SYSTEM", $SecurePassword)
+
+$ConfigurationData = @{
+    AllNodes = @(
+        #AllNodes
+        @{
+            NodeName = "*"
+            PSDscAllowPlainTextPassword = $true
```

Comment:
Add a comment that this is not recommended and point to a reffernce on how to do it correctly.  Such as the blog [want to secure credentials in windows powershell desired state configuration](http://blogs.msdn.com/b/powershell/archive/2014/01/31/want-to-secure-credentials-in-windows-powershell-desired-state-configuration.aspx)